raw/README.md

```diff
@@ -116,4 +116,41 @@ _Steps_:
 3. Repeat this process, except change the dataset to `pcp`:
    ```
    export DATASET=pcp
-   ```
\ No newline at end of file
+   ```
+
+## Downloading raw data from Copernicus automatically on monthly basis using of [Cloud-Run](https://cloud.google.com/run) 
+
+All data can be ingested from [Copernicus](https://cds.climate.copernicus.eu/#!/home) with `google-weather-tools`,
+specifically `weather-dl` (see [weather-tools.readthedocs.io](https://weather-tools.readthedocs.io/)).
+
+_Pre-requisites_:
+
+1. Set up a cloud project with sufficient permissions to use cloud storage (such as
+   [GCS](https://cloud.google.com/storage)) and a Beam runner (such as [Dataflow](https://cloud.google.com/dataflow)).
+   > Note: Other cloud systems should work too, such as S3 and Elastic Map Reduce. However, these are untested. If you
+   > experience an error here, please let us know by [filing an issue](https://github.com/google/weather-tools/issues).
+
+2. Acquire one or more licenses from [Copernicus](https://cds.climate.copernicus.eu/user/register?destination=/api-how-to).
+   > Recommended: Download configs allow users to specify multiple API keys in a single data request via
+   > ["parameter subsections"](https://weather-tools.readthedocs.io/en/latest/Configuration.html#subsections). We
+   > highly recommend that institutions pool licenses together for faster downloads.
+
+3. Create a docker image from the docker file of the [current directory](https://github.com/google-research/arco-era5/tree/main/raw) and push that image in the [GCR](https://cloud.google.com/artifact-registry).
+   > Note: This command will complete the above step. : gcloud builds submit . --tag "gcr.io/PROJECT_NAME/REPOSITORY_NAME:TAG"
```

Comment:
Please add this link here: https://github.com/google/weather-tools/blob/main/Runtime-Container.md.